messages/messages.json

```diff
@@ -28,5 +28,6 @@
   "warnAuth": "Logging in to a business or production org is not recommended on a demo or shared machine. Please run \"sfdx auth:logout --targetusername <your username> --noprompt\" when finished using this org, which is similar to logging out of the org in the browser.\n\nDo you want to authorize this org for use with the Salesforce CLI (y/n)?",
   "noPromptAuth": "do not prompt for auth confirmation in demo mode",
   "disableMasking": "disable masking of user input (for use with problematic terminals)",
-  "clientSecretStdin": "OAuth client secret of personal connected app?"
+  "clientSecretStdin": "OAuth client secret of personal connected app?",
+  "outputUrl": "Output the URL instead of opening it"
```

Comment:
Edit: "output the URL to the terminal instead of opening it in a browser" 

(Our style is to start param descriptions in lower case and not use a period at the end.)

Reply:
Also, we don't usually use camelCase when naming parameters.  So I suggest "--outputurl".  

Is there a short name? If so, how about -o?  

See our style guidelines here: https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_plugins.meta/sfdx_cli_plugins/cli_plugins_style.htm

Reply:
I did notice that and so in login.ts the outputurl isn't camel case and would display as "--outputurl" however it did seem to me that the messages variables were camel case as the others are in this file.

Reply:
Oh!  Sorry, didn't see that.  I'm just the writer :).   So it's all good!